vars/fcosKola.groovy

```diff
@@ -3,8 +3,13 @@ def call(cosaDir = "/srv/fcos") {
     stage('Kola') {
         parallel run: {
             stage("run") {
+                def args = ""
+                def srctests = new File("tests/kola").getAbsolutePath()
+                if (srctests.isDirectory()) {
```

Comment:
This unfortunately doesn't work in Kubernetes pods, e.g. the pipeline has: https://github.com/coreos/fedora-coreos-pipeline/blob/2c6c49599fd242e3ffbc6d4b4b1087fdb731a5ae/utils.groovy#L46. Which, yeah OK let me upstream the `shwrap_rc` function to this repo: https://github.com/coreos/coreos-ci-lib/pull/17. Then we can use it here for this.

I'll double check whether this is fixed, and if not, also upstream `path_exists` as a helper.

Reply:
Ah, nice catch.  OK I took another stab at this but still not tested.